### DISCUSSION TITLE
我续费了340元年费，不知道在哪里能申请开发票

### DISCUSSION THREAD

**Discussion by skyshining** [getlantern/lantern-forum-cn#154] [General]
我续费了340元年费，不知道在哪里能申请开发票

**Comment by MaxMKMax** [comment]
@skyshining 您的帐号有成功被升级到专业版吗? 
您可以检查一下您的邮箱或者到spam folder看看. 系统应该会自动生成发票的.

**Reply by teny416** [reply]
你好，我购买了专业版以后，在邮箱里没有看到发票，请问还可以从什么渠道获取发票？

**Reply by MaxMKMax** [reply]
@teny416 
https://lantern.freshdesk.com/en/support/tickets/new
您好. 请您在这个链接提交问题上去. 谢谢您. 支付系统团队会帮您处理好.
您也检查一下垃圾邮箱.
